planning/behavior_velocity_planner/include/scene_module/occlusion_spot/occlusion_spot_utils.hpp

```diff
@@ -98,17 +96,27 @@ struct PlannerParam
   double angle_thr;      // [rad]
   bool show_debug_grid;  // [-]
 
-  VehicleInfo vehicle_info;
-  EgoVelocity private_road;
-  EgoVelocity public_road;
+  // vehicle info
+  double half_vehicle_width;  // [m]  half vehicle_width from vehicle info
+  double baselink_to_front;   // [m]  wheel_base + front_overhang
+
+  Velocity v;
   Sidewalk sidewalk;
   grid_utils::GridParam grid;
 };
 
+struct SafeMotion
+{
+  double stop_dist;
+  double safe_velocity;
+};
+
 struct ObstacleInfo
 {
+  SafeMotion sm;  // safe motion of velocity and stop point
```

Comment:
The varibale named `sm` is not easy to understand. This should not be omitted even if the code gets longer

Reply:
I stay unchanged for variables which scope is small yet I changed for Class Object inside possible collision which can be  used in many place